contracts/transmuter/configs/Test.sol

```diff
@@ -27,6 +27,8 @@ contract Test {
     ) external {
         LibSetters.setAccessControlManager(_accessControlManager);
 
+        DiamondStorage storage ds = s.diamondStorage();
+        ds.statusReentrant = NOT_ENTERED;
```

Comment:
no need to initialize it no? As it'll go to 1 directly after the first nonReentrant call

Reply:
It's a one shot thing anyway and openzeppelin is doing it